### PR TITLE
Bumps version to 3.0.0

### LINKED ISSUE #699: Preparing next release
I would like to push a new release of Kiwi out to include what it is currently on master, plus the fixes needed to ensure compatibility with Xcode 9 (which we are currently working on).

Initially, the work included in this milestone would be:

- [x] Everything already merged to master since 2.4.0 (see [diff](https://github.com/kiwi-bdd/Kiwi/compare/v2.4.0...master))
- [x] Fixing CI builds (#703)
- [x] Fixing #698: renaming the `any` macro
- [x] Ensure compatibility with Xcode 9 (no warnings)
- [ ] Release new version

I know there were [conversations](https://github.com/kiwi-bdd/Kiwi/issues/662#issuecomment-223775644) for the next release to be v3.0, but it feels like a minor version bump could be more appropriate, since most of the changes added are bug fixes, memory management improvements and other maintenance work. 

@kiwi-bdd/core-team would you be happy for me to tag this release as v2.4.1 or would you rather continue with the plan of naming it v3.0 just for closure?

### PR DESCRIPTION
**Pull request by ecaselles**
Closes #699.

### CODE REVIEW

**Comment by orta** [comment]
You might need CP trunk access?

**Comment by ecaselles** [comment]
@orta Yes please, I was about to ask @sharplet for it 😄 

But if you can grant it too, that would be awesome 👌

**Comment by orta** [comment]
What's your trunk email?

File: Kiwi.podspec [hunk 1]
```diff
@@ -1,19 +1,20 @@
...
-    'Stepan Hruda' => 'stepan.hruda@gmail.com',
-    'Brian Gesiak' => 'modocache@gmail.com',
-    'Adam Sharp'   => 'adsharp@me.com',
+    'Allen Ding'       => 'alding@gmail.com',
+    'Luke Redpath'     => 'luke@lukeredpath.co.uk',
+    'Marin Usalj'      => 'marin2211@gmail.com',
+    'Stepan Hruda'     => 'stepan.hruda@gmail.com',
+    'Brian Gesiak'     => 'modocache@gmail.com',
+    'Adam Sharp'       => 'adsharp@me.com',
+    'Eduardo Caselles' => 'edu@casell.es'
```
Comment (ecaselles) [inline Kiwi.podspec]: This one @orta ;)

Comment (orta) [reply Kiwi.podspec]: hah - oh yeah, good point

**Pull request merged**